packages/security/src/CredentialManagerFactory.ts

```diff
@@ -179,6 +180,36 @@ export class CredentialManagerFactory {
      */
     private static mManager: AbstractCredentialManager;
 
+    /**
+     * Determines the type of CredentialManager to use from passed in type from imperative.json.
+     * If no type is found and manager param is a string, will check if a manager override was
+     * not passed, use the default keytar manager.
+     * @param manager the manager containing the type of CredentialManager to use or string
+     * @returns an Object of AbstractCredentialManager with the CredentialManager to use
+     */
+    private static determineCredentialManagerType(manager: any): any {
```

Comment:
I see the JSDoc explainx the types, but...

should `manager` be `ICredentialManager`?
And should the return value also be updated to `AbstractCredentialManager` (or some other applicable value)?

Again, the same comment about having `any` in source code. 😢